data/helpers.d/network

```diff
@@ -0,0 +1,18 @@
+# Normalize the path syntax
+# Check the slash at the beginning of path and its absence at ending
```

Comment:
"Check" imply it's a check, not a modification.

It could be good to give some examples.